pkgs/development/python-modules/xmljson/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, lxml
+}:
+
+buildPythonPackage rec {
+  pname = "xmljson";
+  version = "0.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b4158e66aa1e62ee39f7f80eb2fe4f767670ba3c0d5de9804420dc53427fdec8";
+  };
+
+  checkInputs = [
+    lxml
+  ];
+  meta = with lib; {
+    description = "Converts XML into dictionary structures and vice-versa";
+    homepage = "https://github.com/sanand0/xmljson";
+    license = with licenses; [ mit ];
+    maintainers = [ maintainers.rakesh4g ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ rakesh4g ];
```

Missing platforms.

Reply:
buildPythonPackage already sets the platform, contributors should not. Unless there are additional limitations.

Reply:
>> maintainers = with maintainers; [ rakesh4g ];

This is done. Platforms not needed.